### PR TITLE
sucessfully fixes some grammar

### LINKED ISSUE #37093: Core R&D Console beeps, "Sucessfully researched Experimental Flight Equipment."
Successfully.

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
closes #37093

### CODE REVIEW

**Comment by RandomMarine** [comment]
Achktuhally this is a spelling fix and not a grammar fix.

**Comment by KorPhaeron** [comment]
why in gods name would you make a changeling that just says "fix" for a typo

**Comment by ExcessiveUseOfCobblestone** [comment]
I height this pr.

**Comment by Tlaltecuhtli** [comment]
muh gbp

**Comment by ExcessiveUseOfCobblestone** [comment]
g/f are 0 anyways sweetie

**Comment by XDTM** [comment]
>why in gods name would you make a changeling that just says "fix" for a typo

New changeling rework?

**Comment by ExcessiveUseOfCobblestone** [comment]
Changelings were a mistake, why do we need to tell them every little thing that changes within the cod?

File: code/modules/clothing/spacesuits/flightsuit.dm [hunk 1]
```diff
@@ -644,14 +644,14 @@
...
-			usermessage("[I] has been sucessfully installed into systems.", mob_override = user)
+			usermessage("[I] has been successfully installed into systems.", mob_override = user)
 			if(user.transferItemToLoc(I, src))
 				if(part_manip)
 					part_manip.forceMove(get_turf(src))
 				part_manip = I
 				changed = TRUE
 		if(istype(S, /obj/item/stock_parts/scanning_module))
-			usermessage("[I] has been sucessfully installed into systems.", mob_override = user)
+			usermessage("[I] has been successfully installed into systems.", mob_override = user)
```
Comment (ShizCalev) [inline code/modules/clothing/spacesuits/flightsuit.dm]: missed it on 661, 668, and 675.

should probably just set the message in a var

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by KorPhaeron** [comment]
@ExcessiveUseOfCobblestone they can absorb and process foreign DNA, why wouldn't they be able to detect changes in a fish?

File: code/modules/clothing/spacesuits/flightsuit.dm [hunk 2]
```diff
@@ -98,6 +98,9 @@
 	var/atom/movable/cached_pull		//recipe for disaster again.
 	var/afterForceMove = FALSE
 	var/datum/component/mobhook
+	
+
+	var/SuccessMessage = " has been successfully installed into systems."
```
Comment (ShizCalev) [inline code/modules/clothing/spacesuits/flightsuit.dm]: just have that in the attackby instead of a new var on the object itself...

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
@KorPhaeron Each time I look at this fred I confuse myself more and more.

File: code/modules/clothing/spacesuits/flightsuit.dm [hunk 3]
```diff
@@ -640,39 +640,40 @@
 	suit = null
 
 /obj/item/device/flightpack/attackby(obj/item/I, mob/user, params)
+	var/SuccessMessage = " has been successfully installed into systems."
```
Comment (vuonojenmustaturska) [inline code/modules/clothing/spacesuits/flightsuit.dm]: make this a `var/const/SuccessMessage` too

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

**Pull request merged**